hpms/src/test/resources/application.hpms.properties

```diff
@@ -9,4 +9,5 @@ spring.jpa.database-platform=org.hibernate.dialect.PostgreSQLDialect
 spring.jpa.hibernate.ddl-auto=none
 spring.liquibase.enabled=true
 
-hpms.base.url=${AB2D_HPMS_URL:#{'http://localhost:8080'}}
\ No newline at end of file
+hpms.base.url=${AB2D_HPMS_URL:#{'http://localhost:8080'}}
+hpms.auth.url=${AB2D_HPMS_AUTH_URL:#{'https://hpmsimpl.cms.gov/api/idm/oauth/token?ACS=wlVuThEThipRlBu37Pra'}}
```

Comment:
You may need to add these to the docker-compose.yml so they make it in there when running from a deployment, you can see examples in there of how it's done in the different environment sections. Same with the HPMS_AUTH_KEY_ID key

Reply:
Done by Will.